Backend/app/controllers/casenote.controller.js

```diff
@@ -0,0 +1,26 @@
+const CaseNote = require('../models/casenote.model');
+const Customer = require('../models/customer.model');
+
+const caseNoteController = {
+   async create(req, res) {
+    try {
+      if (req.body) {
+        const note = new CaseNote(req.body); // TODO: Add data sanitization
+        const case_id = await note.create();
+        const alert_case_id = await Customer.getAlertCaseId(req.body.user_id);
```

Comment:
Consider using Promise.all for fast error/ correct error handling? 
https://stackoverflow.com/questions/45285129/any-difference-between-await-promise-all-and-multiple-await
(I'm personally not super familiar with promise, so I'm just putting this as reference)